teamB/kimSungHoon/routes/postRouter.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const postController = require('../controllers/postController');
+
+const router = express.Router();
+
+router.post('/', postController.createPost);
+
+router.get('/', postController.readingPost);
+
+router.patch('/:postId', postController.updatePost);
+
+router.delete('/:postId', postController.deletePost);
```

Comment:
해당 부분은 코드의 길이가 길어 보일 수 있어 아래와 같이 수정해주세요 !
```suggestion
router.post('/', postController.createPost);
router.get('/', postController.readingPost);
router.patch('/:postId', postController.updatePost);
router.delete('/:postId', postController.deletePost);
```

Reply:
감사합니다 수정하였습니다.